### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by ethanmills**
## Ticket and context
Let's see if Dependabot can help keep our dependencies up to date. We currently only get security updates - let's see if keeping everything more up to date is practical. 

Recommended per https://docs.publishing.service.gov.uk/manual/manage-ruby-dependencies.html

Ticket: N/A

### Is there anything that the code reviewer should know?
Very open to challenges on this. I'm suggesting we try it out, and turn it off if it doesn't work for us

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    open-pull-requests-limit: 5
+  - package-ecosysyem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
+    open-pull-requests-limit: 5
+  - package-ecosystem: "githib-actions"
```
Comment (FriedSock) [inline .github/dependabot.yml:24]: typo?

Comment (ethanmills) [reply .github/dependabot.yml:24]: Yes typo. The actions we depend on are versioned: https://github.com/DFE-Digital/early-careers-framework/blob/develop/.github/workflows/run_task_dev.yml#L20

**Review by kicferk1 — APPROVED** [review]
So do we think this is going to remove the need for snyk? In the long run anyway?

**Comment by ethanmills** [comment]
> So do we think this is going to remove the need for snyk? In the long run anyway?

Possibly, we'll see how it behaves

**Pull request merged**